account_invoice_mgmt_dms/models/account_move.py

```diff
@@ -92,16 +95,28 @@ def open_wizard_dms_extra_file(self):
         wizard = self.open_wizard_dms('account.move.dms.extra.file.wizard')
         wizard['name'] = _('Add Extra Docs')
         return wizard
+    
+    def validation_files(self, dms_file):
+        error = ''            
+        try:
+            self.env['ir.actions.report']._merge_pdfs([io.BytesIO(dms_file.content_binary)])
+        except Exception as e:
+            error = '\n - %s' % dms_file.name
+        return error
 
     def create_dms_file_document_manager(self):
+        validate=self.env.context.get('validate',False)
```

Comment:
According to previous comment, required change:
```suggestion
        validate = self.env.context.get('create_dms_file_document_validate', False)
```
And proper code formatting